demos/sample_azure_iot/sample_azure_iot.c

```diff
@@ -78,26 +81,72 @@
 /**
  * @brief The Telemetry message published in this example.
  */
-#define sampleazureiotMESSAGE                                 "Hello World : %d !"
+// #define sampleazureiotMESSAGE                                 "Hello World : %d !"
+// #define sampleazureiotMESSAGE                                 "{\"temperature\":%f,\"humidity\":%f,\"co2\":%f}"
+
+/**
+ * @brief Telemetry Values
+ */
+// Primary objects
+#define sampleazureiotTELEMETRY_OBJECT_SKID                       ( "skid" )
+#define sampleazureiotTELEMETRY_OBJECT_UNITS                      ( "units" )
+#define sampleazureiotTELEMETRY_OBJECT_EXTERNAL                   ( "environment" )
+#define sampleazureiotTELEMETRY_OBJECT_STATE                      ( "state" )
+#define sampleazureiotTELEMETRY_OBJECT_ERROR_STATE                ( "error-state" )
+// Secondary
+#define sampleazureiotTELEMETRY_OBJECT_UNIT1                      ( "unit1" )
+#define sampleazureiotTELEMETRY_OBJECT_UNIT2                      ( "unit2" )
+#define sampleazureiotTELEMETRY_OBJECT_UNIT3                      ( "unit3" )
+#define sampleazureiotTELEMETRY_OBJECT_PRESSURE                   ( "pressure" )
+#define sampleazureiotTELEMETRY_OBJECT_TEMPERATURE                ( "temperature" )
```

Comment:
We may need 4 temperature values.

Reply:
We have 4 temperature values if you look a bit down. And this is the object,

"temperature": {
                "cartridge1-1": "",
                "cartridge1-2": "",
                "cartridge2-1": "",
                "cartridge2-2": "",
                "cartridge3-1": "",
                "cartridge3-2": "",
                "cartridge4-1": "",
                "cartridge4-2": ""
            },